xlgui/widgets/queue.py

```diff
@@ -38,7 +38,7 @@ def __create_queue_tab_context_menu():
     smi = menu.simple_menu_item
     sep = menu.simple_separator
     items = []
-    items.append(smi('clear', [], None, 'gtk-clear',
+    items.append(smi('clear', [], _("_Clear Queue"), 'edit-clear-all',
```

Comment:
See note above:
`C_lear` is barely readable (letter l is so slim). I chose `_Clear` instead.